### PR TITLE
Describe REC events and weird channel pitch behaviour a bit

### PR DESCRIPTION
**Pull request by praashie**

### CODE REVIEW

File: src/channels/__init__.py [hunk 1]
```diff
@@ -240,24 +240,31 @@ def setChannelPan(index: int, pan: float, pickupMode: int = midi.PIM_None) -> No
...
+            range (usually `-1.0` to `1.0`). Larger values might be reached if the
+            pitch is automated with events, for example.
+
+        * `1`: return the current pitch offset in cents.
+
+            BUG: Official API docs incorrectly state "semitones".
+
+        * `2`: return the current pitch range in semitones.
+
+            BUG: This is not guaranteed to be correct. See `getChannelPitch`.
```
Comment (MiguelGuthridge) [inline src/channels/__init__.py:261]: This seems to be self-referencing?

File: src/channels/__init__.py [hunk 2]
```diff
@@ -507,16 +518,18 @@ def incEventValue(eventId: int, step: int, res: float) -> int:
...
 
 def getRecEventId(index: int) -> int:
-    """Returns recording event ID for channel at `index`.
-    
+    """Return the starting point of REC event IDs for the channel at `index`.
+
+    See `general.processRECEvent` for more information.
+
     HELP WANTED: Honestly REC events are sooooo confusing, and I avoid using 
     them entirely. Can someone else explain them?
```
Comment (MiguelGuthridge) [inline src/channels/__init__.py:526]: If you feel like it's explained sufficiently, feel free to get rid of the `HELP WANTED` flags

**Review by MiguelGuthridge — CHANGES_REQUESTED** [review]
Looks pretty good to me! There's a couple of things that could be improved but otherwise this will be awesome!

File: src/channels/__init__.py (hunk 1 above)
Comment (praashie) [reply src/channels/__init__.py]: Nice catch, `setChannelPitch` is the actual intended reference.

File: src/channels/__init__.py (hunk 2 above)
Comment (praashie) [reply src/channels/__init__.py]: We definitely need to write proper examples and tutorials regarding this topic, but in the scope of docstrings, my explanation is probably sufficient for now.

**Review by MiguelGuthridge — APPROVED** [review]
Looks good to me!

**Pull request merged**